.github/workflows/nightly.yml

```diff
@@ -0,0 +1,40 @@
+name: Nightly
+
+on:
+  workflow_dispatch:
+
+jobs:
+  image:
+    name: Publish Image
+    environment: Development
+    runs-on: Ubuntu-20.04
+    env:
+      APPLICATION: fableous
+    strategy:
+      matrix:
+        service: [be, fe]
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Set Development Version
+        run: echo "RELEASE_VERSION=v0.0.0-development" >> $GITHUB_ENV
+      - name: Provide Env File
+        run: |
```

Comment:
is the char '|' intentional here? if so whats its purpose?

Reply:
Multiline string in YAML